### ISSUE TITLE
lemonbar binding problems

### ISSUE THREAD

**Issue by priestoferis** [powerline/powerline#2052]
I'm using Ubuntu 18.04 with regolith, and I had two problems with using lemonbar with powerline. One was related to lemonbar in the repo not finding the fonts (`apt install lemonbar fonts-powerline`), so I needed to use a different [fork](https://github.com/krypt-n/bar) which found it. But I still have some problems. Running `python /usr/local/lib/python2.7/dist-packages/powerline/bindings/lemonbar/powerline-lemonbar.py -i3 - -f "Source Code Pro for Powerline Medium 13" -f "Font Awesome"` results in powerline looking like this:

![image](https://user-images.githubusercontent.com/6343487/69563557-a0d28280-0fb1-11ea-8a14-1f0f6524dce8.png)

while running the same powerline config in `i3bar` looks like this:

![image](https://user-images.githubusercontent.com/6343487/69563352-3ae5fb00-0fb1-11ea-94ea-99ab7d736472.png)

notice the arrow(?) differences.

**Comment by BlitzKraft** [comment]
I feel this might be a config or environment error. The font is not source code pro. 

Lemonbar by default, doesn't seem to support xft. (The specified font in the -f option). There seems to be a separate fork of lemonbar with xft support. Using that should fix your problem.

**Comment by priestoferis** [comment]
Hmm, you are right, it seems I thought that the powerline font was a totally different font, but no. 

Anyhow 
```
python /usr/local/lib/python2.7/dist-packages/powerline/bindings/lemonbar/powerline-lemonbar.py -i3 - -f "Source Code Pro Medium" -f "Font Awesome"
``` 
works, so indeed it was a fonts issue. The only thing that doesn't work now is strangely is that with `lemonbar` the pause sign doesn't show up, only the play sign, while that works in `i3bar` (with the "new" font).

**Comment by BlitzKraft** [comment]
It's possible the pause glyphs are different. This might have something to do with font configuration on your system. `fc-list` command will list out all fonts available. Check if the powerline fonts are listed. That command can output a lot, pipe it through grep or/and less. 

If they are not found in the fc-list output, then you might need to manually copy the font to one of the directories where fc scans. This list usually includes `~/.fonts` and you can create the directory if needed.

**Comment by priestoferis** [comment]
I ended up specifying the state symbols in the powerline config file:

```
          "state_symbols": {
                       "fallback": "f",
                       "pause": "\uf04c",
                       "stop": "\uf04d",
                       "play": "\uf04b"
                    }
```

and now it works. As a note, the `fonts-powerline` package didn't seem to work at all, I manually installed the fonts from the repo, because `fc-list` didn't show half of the fonts that were supposed to be installed, but this was not the solution for the state symbols. Thank you for your help!
